pkgs/by-name/us/ustreamer/package.nix

```
@@ -16,8 +16,11 @@
   nixosTests,
   systemdLibs,
   which,
+  python3,
+  python3Packages,
```

```suggestion
  python3Packages,
```
adding both makes it easy to miss one and mix python versions which must be avoided at all cost